group-coordinator/src/main/java/org/apache/kafka/coordinator/group/consumer/ConsumerGroupMember.java

```diff
@@ -545,6 +547,32 @@ public String currentAssignmentSummary() {
             ')';
     }
 
+    public ConsumerGroupDescribeResponseData.Member asConsumerGroupDescribeMember(Assignment targetAssignment) {
```

Comment:
Instead of passing the `targetAssignment` here, I wonder if we could just set it on the caller side. Have you considered this?

Reply:
It would be possible of course.. but we would have redundant code to map from a Map of partitions to a List of TopicPartitions in ConsumerGroupMember.java and ConsumerGroup.java
I think its better the way it is, pass the targetAssignment to ConsumerGroupMember.java and use the private static method topicPartitionsFromMap

let me know what you think

Reply:
I see your point. I don't have a strong opinion on this so I let you choose what you think is best.

Reply:
I will keep it the way it is then